keyboards/exclusive/e7v1se/readme.md

```diff
@@ -0,0 +1,15 @@
+# e7v1se
+
+![e7v1se](https://imgur.com/a/6IjA0cC)
```

Comment:
```suggestion
![e7v1se](https://i.imgur.com/ksbDEYh.jpg)
```
This needs to be the direct link to the image, otherwise it won't render.

Reply:
Updated!